src/bundles/ApplicationsAdmin/components/AppDiff/AppDiff.css

```diff
@@ -0,0 +1,18 @@
+tr.row {
+	border-top-color: black;
+	border-top-style: solid;
+	border-top-width: 2px;
+}
+
+td.column {
+	width: 25%;
+	padding-right: 1rem;
+	padding-top: 1rem;
+	padding-bottom: 2rem;
```

Comment:
This can be shortened to `padding: 1rem 1rem 2rem 0;`

Reply:
fixed